### ISSUE TITLE
Request for Fixed Y scale on serial plotter

### ISSUE THREAD

**Issue by Bort-Plate** [arduino/arduino-serial-plotter-webapp#26]
With values that constantly change, the range of the plot will shrink and expand, making it incredibly difficult to judge the amplitude of a plot without constantly re-checking the Y-axis (of which is too small to read on my 4K screen anyway). A 3 inch peak may have a max value of 500, and 10 seconds later, another 3 inch peak will have a value of 25! 

By having a selectable minimum and maximum, each peak on the plot can be visually compared properly.

**Comment by karlTH** [comment]
serial plotter is an awesone tools
a csv export will be really good.
thanks a lot for the work.
i m also agree with cousteaulecommandant

**Comment by phoenixperry** [comment]
I am going to agree that this is really difficult to work with and could use a fix.

**Comment by alecokas** [comment]
I agree with @MickMad. Has any progress been made on this issue? I think it would be a useful addition.

**Comment by steeley** [comment]
yep, 3 years on and still no option....
Plotter is very useful, but on/off autoscale should have been a fundamental feature.
I had to hack  together my own plotter as part of a desktop app, but would be easier
in Arduino IDE.

**Comment by gohai** [comment]
The ability to have a fixed scaling on the Y-axis is direly needed, and should be the default. Workarounds such as [this one](https://forum.arduino.cc/t/serial-plotter-stabe-y-axis/558626/4) are possible but add unneeded complexity in a learning situation.
